rating_api/routes/comment.py

```diff
@@ -20,25 +20,40 @@
 @comment.post("", response_model=CommentGet)
 async def create_comment(lecturer_id: int, comment_info: CommentPost, user=Depends(UnionAuth())) -> CommentGet:
     """
+    Scopes: `["rating.comment.review"]`
     Создает комментарий к преподавателю в базе данных RatingAPI
     Для создания комментария нужно быть авторизованным
     """
     lecturer = Lecturer.get(session=db.session, id=lecturer_id)
     if not lecturer:
         raise ObjectNotFound(Lecturer, lecturer_id)
 
+    if comment_info.create_ts or comment_info.update_ts:
+        if not "rating.comment.review" in [scope['name'] for scope in user.get('session_scopes')]:
+            raise ForbiddenAction(Comment)
+        else:
+            if not comment_info.create_ts:
+                comment_info.create_ts = datetime.datetime.utcnow()
```

Comment:
`comment_info.create_ts = datetime.datetime.utcnow() if comment_info.create_ts is None`